packages/devtools_app/lib/src/screens/performance/performance_screen.dart

```diff
@@ -131,8 +133,60 @@ class WebPerformanceScreenBody extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return const Center(
-      child: Text('TODO: add instructions for using Chrome DevTools'),
+    final isFlutterWebApp =
+        serviceManager.connectedApp?.isFlutterWebAppNow ?? false;
+    return Markdown(
+      data: isFlutterWebApp ? flutterWebInstructionsMd : dartWebInstructionsMd,
+      onTapLink: (_, url, __) {
+        if (url != null) {
+          unawaited(
+            launchUrl(
+              Uri.parse(url),
+            ),
+          );
+        }
+      },
     );
   }
 }
+
+const timelineLink =
+    'https://api.flutter.dev/flutter/dart-developer/Timeline-class.html';
+const timelineTaskLink =
+    'https://api.flutter.dev/flutter/dart-developer/TimelineTask-class.html';
+const debugBuildsLink =
+    'https://api.flutter.dev/flutter/widgets/debugProfileBuildsEnabled.html';
+const profileModeLink = 'https://docs.flutter.dev/testing/build-modes#profile';
+const performancePanelLink =
+    'https://developer.chrome.com/docs/devtools/performance';
+
+const flutterWebInstructionsMd = '''
+# How to use Chrome DevTools for performance profiling
+
+The timeline events emitted by the Flutter framework as it works to build
+frames, draw scenes, and track other activity such as HTTP request timings and
+garbage collection are exposed in the Chrome DevTools performance panel.
+
+You can also emit your own timeline events using the `dart:developer` 
+[Timeline]($timelineLink) and [TimelineTask]($timelineTaskLink) APIs for further
+performance analysis.
+
+## Instructions:
+
+1. *[Optional]* Set [debugProfileBuildsEnabled]($debugBuildsLink) to true to 
+track your app's widget builds.
+2. Run your Flutter web app in [profile mode]($profileModeLink).
+3. Open up the [Chrome DevTools' Performance panel]($performancePanelLink) for
+your application, and start recording to capture timeline events.
+''';
+
+const dartWebInstructionsMd = '''
```

Comment:
for Dart web app users, should we also recommend that they build their app in AOT mode?

Reply:
There's different ways to do so (depending on if they are using webdev, acx, etc) so leaving as is for now